### PR TITLE
Fixes for #65, #108, and #102

### PR DESCRIPTION
**Pull request by sgnn7**
- Fixes cleanup of most cases of exits on Linux (#108)
- Upgrades to Golang 1.13 (#102)
- Statically builds Linux binaries (#65)

[Jenkins Build](https://jenkins.conjur.net/job/cyberark--summon/job/106-tempfile-cleanup/)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [v0.8.0](https://github.com/cyberark/summon/releases/tag/v0.8.0) - 2019-09-20
+
+### Changed
+- To ensure cleanup of files on non-windows platforms we now remain resident
+  until the child is killed or it exits [#106](https://github.com/cyberark/summon/issues/106)
+- Updated base Golang version to 1.13
+- Made Linux bulds create static binaries [#65](https://github.com/cyberark/summon/issues/65)
```
Comment (izgeri) [inline CHANGELOG.md]:
```suggestion
- Made Linux builds create static binaries [#65](https://github.com/cyberark/summon/issues/65)
```

File: .goreleaser.yml [hunk 2]
```diff
@@ -6,18 +6,34 @@ before:
   - go mod download
 
 builds:
-- binary: summon
+- id: summon
+  binary: summon
   env:
   - CGO_ENABLED=0
+  flags:
+  - -tags=netgo
```
Comment (izgeri) [inline .goreleaser.yml]: why are you using this flag?

File: .goreleaser.yml [hunk 3]
```diff
@@ -6,18 +6,34 @@ before:
...
+  env:
+  - CGO_ENABLED=1
+  flags:
+  - -tags=netgo
+  goos:
   - linux
   goarch:
   - amd64
-  ldflags: []
+  ldflags: -s -w -linkmode external -extldflags "-static"
```
Comment (izgeri) [inline .goreleaser.yml]: that's a bunch of new flags for linux - how did you land on this set?

Comment (izgeri) [reply .goreleaser.yml]: also, why does linux need CGO?

**Comment by marco-m** [comment]
In my understanding, this fixes #106, more than #108.

**Pull request merged**